tests/unit/records/serializers/literature/common/test_reference_item.py

```diff
@@ -107,3 +108,100 @@ def test_returns_non_empty_fields_if_some_fields_missing():
     result = schema.dumps(dump).data
 
     assert expected == json.loads(result)
+
+
+def test_returns_no_misc_if_title_persent():
+    schema = ReferenceItemSchemaV1()
+    dump = {
+        'reference': {
+            'title': {
+                'title': 'Jessica Jones',
+            },
+            'misc': [
+                'A Misc',
+            ]
+        }
+    }
+    expected = {
+        'titles': [
+            {
+                'title': 'Jessica Jones'
+            }
+        ]
+    }
+    result = schema.dumps(dump).data
+
+    assert expected == json.loads(result)
+
+
+def test_returns_no_misc_if_titles_persent_in_the_resolved_record():
+    schema = ReferenceItemSchemaV1()
+    dump = {
+        'record': {
+          '$ref': 'http://localhost:5000/api/literature/1472986',
+        },
+        'reference': {
+            'label': '123',
+            'title': {
+                'title', 'ABCCCCC',
+            },
+            'misc': 'A misc',
+        }
+    }
+    result = schema.dumps(dump).data
+
+    assert 'misc' not in json.loads(result)
+
+
+def test_returns_only_first_misc():
+    schema = ReferenceItemSchemaV1()
+    dump = {
+        'reference': {
+            'label': '123',
+            'misc': [
+                'A Misc',
+                'Another Misc',
+            ]
+        }
+    }
+    expected = {
+        'label': '123',
+        'misc': 'A Misc',
+    }
+    result = schema.dumps(dump).data
+
+    assert expected == json.loads(result)
+
+
+def test_references_satisfies_schema():
+    schema = load_schema('hep')
+    subschema = schema['properties']['references']
+
+    data = {
+        'references': [
+            {
+                'reference': {
+                    'authors':[
+                        {
+                            'full_name': 'Sakharov, A.D.'
+                        }
+                    ],
+                    'label': '1',
+                    'misc': [
+                        'Violation of CP Invariance, C Asymmetry, and Baryon Asymmetry of the Universe [Usp. Fiz. Nauk161,61(1991)]'
+                    ],
+                    'publication_info': {
+                        'artid': '32',
+                        'journal_title': 'Pisma Zh.Eksp.Teor.Fiz.',
+                        'journal_volume': '5',
+                        'page_start': '32',
+                        'year': 1967
+                    }
+                },
+                'record': {
+                    '$ref': 'http://localhost:5000/api/literature/1472986',
+                }, 
+            }
+        ]
+    }
+    assert validate(data['references'], subschema) is None
```

Comment:
you have to validate for each test case, no need to add a separate test for the schema